### ISSUE TITLE
Inconsistency in Action info API

### ISSUE THREAD

**Issue by tuler** [openwhisk/openwhisk#828]
Consider the following API, and one example using the default whisk.system namespace:
`GET /namespaces/{namespace}/actions`
`GET /namespaces/whisk.system/actions`

The API returns an array like this:
```
[
  {
    name: 'wordCount',
    ...
    namespace: 'whisk.system/samples'
  }, ...
]
```

Then we have the following API
`GET /namespaces/{namespace}/actions/{actionName}`

If we take the `namespace` value of the action and `name` value of the action we would create the following REST call:
`GET /namespaces/whisk.system/samples/actions/wordCount`

But this URL is invalid. The URL that does work is:
`GET /namespaces/whisk.system/actions/samples/wordCount`

I understand `samples` is not the `namespace` and neither the `actionName`, it's the `package`.
So I would expect the following action structure in the first place:

```
[
  {
    name: 'wordCount',
    ...
    namespace: 'whisk.system',
    package: 'samples'
  }, ...
]
```

**Comment by rabbah** [comment]
Agree. Related to #219. In v2 of the API we will consolidate the packages API and also cleanup the activations API. Will open issues for v2 very soon.
